dashboard/app/assets/javascripts/soundEffects.js

```diff
@@ -227,24 +227,35 @@ Sound.prototype.newPlayableBufferSource = function(buffer, options) {
   var newSound = this.audioContext.createBufferSource();
 
   // Older versions of chrome call this createGainNode instead of createGain
-  var gainNode;
   if (this.audioContext.createGain) {
-    gainNode = this.audioContext.createGain();
+    this.gainNode = this.audioContext.createGain();
   } else if (this.audioContext.createGainNode) {
-    gainNode = this.audioContext.createGainNode();
+    this.gainNode = this.audioContext.createGainNode();
   } else {
     return null;
   }
 
   newSound.buffer = buffer;
   newSound.loop = !!options.loop;
-  newSound.connect(gainNode);
-  gainNode.connect(this.audioContext.destination);
-  gainNode.gain.value = typeof options.volume === "undefined" ? 1 : options.volume;
+  newSound.connect(this.gainNode);
+  this.gainNode.connect(this.audioContext.destination);
+  var startingVolume = typeof options.volume === "undefined" ? 1 : options.volume;
+  this.gainNode.gain.setValueAtTime(startingVolume, this.audioContext.currentTime);
```

Comment:
While trying to implement cross-fade I learned that `gain.linearRampToValueAtTime` doesn't work if you never set a value with one of the other `AudioParam` methods - setting the gain directly here wasn't doing it, so the linear ramp having no point of reference would just snap to the new value.

If you are nervous about this change, I can always leave it as-is and change the crossfade to do
```
setValueAtTime(1, currentTime);
linearRampToValueAtTime(0, currentTime+duration);
```